index.rst

```diff
@@ -48,10 +48,205 @@
+
+
+Design-neutral Requirements
+===========================
+
+Rehardless of whether the ETL or new proposed architecture is adopted, the eventual architecture needs to show how it can meet satisfy the following requirments and use cases.  
+
+
+Availability of the DM-EFD capabilities
+----------------------------------------
+
+If, as anticipated, DM tooling is the primary of interface to EFD data for anyone beyond hardware-level engineers, availability of those services will be important to operational staff in Chile and the US, as well as to science users. It is therefore a requirement that the entire architecture is structured so that sandbox deployments, rolling upgrades and carefully coordinated downtime are the norm for routine operations. 
+
+Interfaces
+----------
+
+Data should be available via TAP/ADQL services as other data sources available to the Science Platform.
+
+The interface to the Science Platform should be deployment- and time-invariant: the same notebook accessing EFD data should run without modifications on the day in Chile and a month later at the LDF.
+
+A syntactic sugar to make access to EFD data more pythonic from the notebook (and to shield the user from schema implementation details) has been requested. Here is an example of how a notebook user could obtain statistics on the M1/M3 temperature sensors::
+
+  import numpy as np
+  import lsst.efd as efd
+  ...
+
+  # Get the temperatures in one go 
+  envtemp = efd.get("m1m3.actuators.envtemp")
+  stdev = np.std(envtemp)
+  mean = np.mean(envtemp)
+  print(f"temperature = {mean} +/- {stdev} K")
+
+
+
+Aggregation
+-----------
+
+The purpose of aggregation is both to reduce volume on high-frequency telemetry data and to increase the signal-to-noise of busy telemetry. Science Platform users are generally interested in events at the same order of cadence as a camera exposure; therefore we propose that all telemetry data sampled with a frequency higher than 1Hz is (1) sampled at 1Hz and (2) aggregated to 1Hz using these generic statistics:
+
+* Max
+
+* Min
+
+* Mean
+
+* Median
+
+* Standard Deviation
+
+For command streams, no aggregation should be done.
+
+For event streams we propose that using Kafka we sum repeats of the
+same messages within the 1Hz window (eg if the M1M3 subsystem issues a
+limitError even 100 times in the last second, a repeat counter of 100
+is stored with the event).
+
+
+Latency
+-------
+
+Latency should be addressed in two parts:
+
+1. Persistence latency -- This is the latency between an even being published on the DDS to that event showing up as an aggregated quantity in the DM EFD.  This latency should be equal to or less than the time to take and reduce a single raft of data on a parallel reduction system.  This puts an upper bound on the sampling rate for the aggregated event streams. For Auxillary data, lower latencies are required; for example CBP data has been requested to be available at 1-second scale latencies. 
```

Comment:
You need to answer "why" here, rather than just asserting.

Reply:
Don't the use cases answer the "why"?